deltachat-ios/Controller/AllMediaViewController.swift

```diff
@@ -78,6 +86,11 @@ class AllMediaViewController: UIPageViewController {
         }
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationItem.rightBarButtonItem = chatId == 0 && UserDefaults.standard.bool(forKey: "location_streaming") ? mapButton : nil
```

Comment:
What's the reason that this is a button in the `NavigationBar` and not another option in the segmented control?

Reply:
pragmatism :) the reason is that we want to reuse the WebxdcViewController, at least for now, and not deal with fragments or whatnot. but i also find it not too bad meanwhile